doc/objectives.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Objectives.md"
```

Comment:
No ".md" either. This is the title of the document.

Reply:
Ok understood.